---
input_bag: "bags/reconstruction/part1_x6_y4_theta33_cpitch10_.bag"
output_bag_dir: "bags/reconstruction/part1_x6_y4_theta33_cpitch10_"

input_dir: "bags/reconstruction/part1_x6_y4_theta33_cpitch10_"
output_dir: "bags/reconstruction/part1_x6_y4_theta33_cpitch10_"

num_bag_clouds: 20

save_output: false
translate_output: false
automatic_bounds: false
use_clustering: true
new_scan: false

transform_input: true
pre_rotation:    [-0.0 5, 3.17, 0.0] #
pre_translation: [0.0, 0.0, 0.8128] # 32.0*0.0254 

bounding_box: [-0.4, 0.5, 0.15, 0.5, 0.91, 2.0] # bounding box limits
voxel_size: 0.005

